va_explorer/templates/partials/_fab_nav.html

```diff
@@ -0,0 +1,7 @@
+<div class="backdrop"></div>
+<div class="fab child to-top" data-subitem="3"><i class="fa fa-caret-up"></i></div>
+<div class="fab child to-bottom" data-subitem="2"><i class="fa fa-caret-down"></i></div>
+<a href="{{request.META.HTTP_REFERER}}">
```

Comment:
We should probably not just blindly redirect back to HTTP referrer.

The Referer header can be spoofed (and possibly exploited), and may not always be present.

Might be better to sanitize it somehow and make sure the referrer (1) exists (2) is a valid location on the current host and not somewhere on a random domain.

Something similar to how Django sanitizes the "next" URL when logging in:

https://github.com/django/django/blob/5fcfe5361e5b8c9738b1ee4c1e9a6f293a7dda40/django/contrib/auth/views.py#L69-L80